design/ceph/object/store.md

```diff
@@ -119,3 +119,39 @@ Specifying this section also ensures that the pool section in the ceph-object-zo
   zone:
     name: "name"
 ```
+
+### LDAP
+
+The Ceph Object Gateway supports integrating with LDAP for authenticating and creating users, please refer [here](https://docs.ceph.com/docs/master/radosgw/ldap-auth/). This means that the `ceph object store user` is also required to be part of groups in the LDAP server otherwise, authentication will fail. The following settings need to be configured in the RGW server:
```

Comment:
> This means that the `ceph object store user` is also required to be part of groups in the LDAP server otherwise, authentication will fail.

Explain this more. What does this mean for CephObjectStoreUser resources and/or OBCs?

Reply:
AFAIK, local users will still work as long as they do not conflict with ldap users.  I don't know which backend has priority if there is a conflict. I spent a few minutes scanning through and don't understand how `LocalEngine` and `LDAPEngine` hook in to the authentication flow. https://github.com/ceph/ceph/tree/02948ff7c61edca8b5e80e22806b69526370a066/src/rgw and don't see